backend/transcript/views.py

```diff
@@ -302,8 +296,9 @@ def save_transcription(request):
         video = Video.objects.get(pk=video_id)
 
         # If transcript doesn't exist then save a new transcript object
+        # TODO: Check if this is the expected transcript type?
         transcript_obj = Transcript(
-            transcript_type=MANUALLY_CREATED,
+            transcript_type=UPDATED_MACHINE_GENERATED,
```

Comment:
@reviwers please check if this is the expected transcript type for a new transcript being created?